forecasting/src/autogluon/forecasting/models/gluonts_model/abstract_gluonts/model_trial.py

```diff
@@ -0,0 +1,60 @@
+import os
+import time
+import logging
+from autogluon.core.utils.loaders import load_pkl
+from autogluon.core.utils.exceptions import TimeLimitExceeded
+from autogluon.core import args
+
+logger = logging.getLogger(__name__)
+
+
+@args()
+def model_trial(args, reporter):
+    try:
+        model, util_args, params = prepare_inputs(args)
+
+        train_data = load_pkl.load(util_args.directory + util_args.train_data_path)
+        val_data = load_pkl.load(util_args.directory + util_args.val_data_path)
+        eval_metric = model.eval_metric
+        model = fit_and_save_model(model, params, train_data, val_data, eval_metric, util_args.time_start,
+                                   time_limit=util_args.get('time_limit', None))
+
+    except Exception as e:
+        if not isinstance(e, TimeLimitExceeded):
+            logger.exception(e, exc_info=True)
+        reporter.terminate()
+    else:
```

Comment:
Is this else for `if not isinstance(e, TimeLimitExceeded):` ? If yes, add more space before `else:`

Reply:
> Is this else for `if not isinstance(e, TimeLimitExceeded):` ? If yes, add more space before `else:`

No, that else is for try, so if there is no error, then reporter will work.